virtual-desktop/pluginDefinition.json

```diff
@@ -1,5 +1,5 @@
 {
-  "identifier": "org.zowe.zlux.ng2desktop",
+  "identifier": "org.zowe.zlux.ivydesktop",
```

Comment:
There is a lot of places we now need to change this:
zlux-app-manager\virtual-desktop\src\app\shared\logger.ts
zlux-app-manager\virtual-desktop\src\app\start-url-manager\start-url-manager.service.ts
zlux-app-manager\virtual-desktop\src\app\window-manager\mvd-window-manager\personalization-panel\personalization-panel.component.ts possibly? 
probably others

so do a text search & see if it makes sense to change there too

Reply:
logger & start-url-manager = done
others = no, they seem to have different ids. "org.zowe.zlux.ng2desktop.settings" is not "org.zowe.zlux.ng2desktop". Changing it could have consequences like breaking saved settings.
That may be desirable at some point in the future but I don't see the case for it now.